tests/test_block_async_io.py

```diff
@@ -17,6 +17,11 @@
 from .common import extract_stack_to_frame
 
 
+@pytest.fixture(autouse=True)
+def unpatch_block_async_io(disable_block_async_io_after):
```

Comment:
```suggestion
@pytest.mark.usefixtures("disable_block_async_io")
def disable_block_async_io():
```

Reply:
this doesn't seem to work in combination with autouse

Reply:
https://docs.astral.sh/ruff/rules/pytest-erroneous-use-fixtures-on-fixture/